src/deluge/gui/views/instrument_clip_view.cpp

```diff
@@ -3486,11 +3486,15 @@ void InstrumentClipView::someAuditioningHasEnded(bool recalculateLastAuditionedN
 		exitUIMode(UI_MODE_AUDITIONING);
 		auditioningSilently = false;
 
```

Comment:
this function is called when an audition pad is let go. if a current pad is being held in the automation view, we don't want the auditioning ended code to override the value popup that is currently being displayed. So I added a check: if you're in the automation view, and a pad is currently being held, do not reset the display.